lib/node_modules/@stdlib/math/generics/random/sample/README.md

```diff
@@ -98,7 +98,8 @@ var out = mysample( [ 0, 1, 2, 3, 4 ] )
 // e.g., returns [ 4, 3, 4, 4 ]
 ```
 
-If provided an array-like object `pool`, the created function will be bound to sample from the supplied object.
+If provided an array-like object `pool`, the returned function will always sample from the supplied object.
+
 
 ``` javascript
 var mysample = sample.factory( [1,2,3,4,5,6] );
```

Comment:
Let's go ahead and make all `array` presentations consistent throughout the README.

```
[ 1, 2, 3, 4, 5, 6 ]
```

This also applies to the `factory` examples below.